### ISSUE TITLE
Most Updated/Maintained Fork?

### ISSUE THREAD

**Issue by jbeacham** [squaredup/HubSpot.NET#117]
Really is unfortunate that there is no maintained C# Library for all of hubspot's new APIs/features. Does anyone know of a better, updated Fork? By going to the Insights tab I can see the progress of several other forks by date, but it is hard to tell which ones actually have more options, v3 calls, etc.

**Comment by cgwhouse** [comment]
@jbeacham I'm needing to write a HubSpot integration myself, and wondering if you have any further insights (ha!) on this issue... found any good forks, or alternate .NET clients to use? I was led to this particular repo by [this answer on HubSpot Community](https://community.hubspot.com/t5/APIs-Integrations/Is-there-a-net-wrapper-for-hubspot-API/m-p/251100), but I don't necessarily feel comfortable using this library for a new project in its current state.

**Comment by LarsPosthuma** [comment]
Start using: https://github.com/Chinchilla-Software-Com/HubSpot.NET

**Comment by Psypher9** [comment]
You're right, it's been a while. We need to start back up. We have plans to move off of RestSharp and make this project a lot lighter weight.

**Comment by cgwhouse** [comment]
Thanks to you both for weighing in! @LarsPosthuma @Psypher9 I think I have some good options here.

Would be thrilled to see things pick back up with official .NET support as well!
